spec/generateRequestObj.spec.js

```diff
@@ -247,6 +248,55 @@ describe('generateRequestObj', function () {
                 expect(noMatch).to.equal(false);
             });
         });
+
+        it('Request should have digitrust params', function() {
+
+            browser.topWindow.DigiTrust = { 
+              getUser: function (key) {
+                  return {
+                    success: true,
+                    identity: {
+                      privacy: {optout: false},
+                      id: 'testId',
+                      keyv: 4
+                  }
+              }
+            }}
+            
+            requestObject = partnerModule.generateRequestObj(returnParcels);
+            var payload = requestObject.data;
+            expect(payload.user.eids).to.deep.equal([{
+              'source': 'digitru.st',
+              'uids': [{
+                'id': 'testId',
+                'atype': 1,
+                'ext': {
+                     'keyv': 4
+                }
+              }]
+            }]);
+            delete browser.topWindow.DigiTrust;
+          });
+
+          it('Request should not have digitrust params when user has optout', function() {
+
+            browser.topWindow.DigiTrust = { 
+              getUser: function () {
+                  return {
+                    success: true,
```

Comment:
success: false from Digitrust in response case is not handled here, refer to Prebid adapter implementation for more test cases.

Reply:
Done